pyproject.toml

```
@@ -28,37 +28,37 @@ packages = [{include = "lerobot"}]
 
 [tool.poetry.dependencies]
 python = ">=3.10,<3.13"
-termcolor = "^2.4.0"
-omegaconf = "^2.3.0"
-wandb = "^0.16.3"
-imageio = {extras = ["ffmpeg"], version = "^2.34.0"}
-gdown = "^5.1.0"
-hydra-core = "^1.3.2"
-einops = "^0.8.0"
-pymunk = "^6.6.0"
-zarr = "^2.17.0"
-numba = "^0.59.0"
+termcolor = ">=2.4.0"
+omegaconf = ">=2.3.0"
+wandb = ">=0.16.3"
+imageio = {extras = ["ffmpeg"], version = ">=2.34.0"}
+gdown = ">=5.1.0"
+hydra-core = ">=1.3.2"
+einops = ">=0.8.0"
+pymunk = ">=6.6.0"
+zarr = ">=2.17.0"
+numba = ">=0.59.0"
 torch = "^2.2.1"
-opencv-python = "^4.9.0.80"
-diffusers = "^0.27.2"
-torchvision = "^0.18.0"
-h5py = "^3.10.0"
-huggingface-hub = "^0.21.4"
+opencv-python = ">=4.9.0.80"
```

Thanks a lot for this! I have a minor suggestion:

```suggestion
opencv-python = ">=4.9.0"
```
Do you think it is ok to just required 4.9.0 as minimum version instead of 4.9.0.80 ? The main problem is that PyPI `opencv-python` package are not managed and uploaded by the core opencv team, so the real opencv version is `4.9.0`, while the `.80` is added by the team uploading the `opencv-python` wheels on PyPI. The problem is that if you are building opencv python bindings from source, your `opencv-python` version will be `4.9.0` , not `4.9.0.80`. While it is possible to always patch opencv-python's pip metadata, as long as you are consuming PyPI releases of opencv-python, `">=4.9.0.80"` and `">=4.9.0` should be equivalent.

---

xref: https://github.com/conda-forge/opencv-feedstock/issues/408

---

I see, thanks for the heads up.
I'm pushing the changes on pusht and here.

---

Great, thanks!